tests/daemon/unit/api/flow3.yml

```diff
@@ -0,0 +1,20 @@
+jtype: Flow
+with:
```

Comment:
Lets add a test `--grpc-data-requests` and no compound pod?

Reply:
Done